am-utils-6.2 - allow autoconf 2.72

From: Ian Kent <raven@themaw.net>

am-utils creates configure at build time and requires versions of autoconf
that are known to work ok.

Allow autoconf version 2.72 to get a build.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 bootstrap |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/bootstrap b/bootstrap
index d454c63..55b2084 100755
--- a/bootstrap
+++ b/bootstrap
@@ -10,7 +10,8 @@ validateversion() {
     case "$v" in
     *2.69)	;;
     *2.71)	;;
-    *)		echo "am-utils requires autoconf 2.69 or 2.71, you have:"
+    *2.72)	;;
+    *)		echo "am-utils requires autoconf 2.69, 2.71 or 2.72, you have:"
 		echo "	$v"
 		exit 1;;
     esac
